src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -374,6 +409,17 @@ class Config:
         return result
 
 
+class iSCSITargetAuthCredentialModel(sa.Model):
+    __tablename__ = 'services_iscsitargetauthcredential'
+
+    id = sa.Column(sa.Integer(), primary_key=True)
+    iscsi_target_auth_tag = sa.Column(sa.Integer(), default=1)
+    iscsi_target_auth_user = sa.Column(sa.String(120))
+    iscsi_target_auth_secret = sa.Column(sa.String(120), default=None)
```

Comment:
Do we have to explicitly define `nullable` here or just specifying the `default` as `None` is good ?